packages/web/src/scss/tools/_form-fields.scss

```diff
@@ -35,6 +37,11 @@
 @mixin inline-field-root() {
     display: inline-flex;
     margin-block: form-fields-theme.$spacing;
+    cursor: pointer;
```

Comment:
This (and the one below) should be stored in `settings` too. The question is how to name it… Maybe like this?

```scss
$interaction-form-fields: pointer;
$interaction-disabled: disabled;
```

That would allow us to extend it later:

```scss
$interaction-button-links: pointer;
$interaction-draggable: grab;
// …
```

(Maybe the word `interaction` is needless.)

Reply:
So I stored all cursors settings there.